quinn-proto/src/connection/mod.rs

```diff
@@ -1807,7 +1793,7 @@ where
         ecn: Option<EcnCodepoint>,
         data: BytesMut,
     ) {
-        self.total_recvd = self.total_recvd.wrapping_add(data.len() as u64);
+        self.path.total_recvd = self.path.total_recvd.saturating_add(data.len() as u64);
```

Comment:
Will take a long time to saturate :-) 

But fair to be correct

Reply:
Yeah, this is mostly a clarification of intent thing.